builtin/game/chatcommands.lua

```diff
@@ -576,14 +579,25 @@ local function handle_give_command(cmd, giver, receiver, stackstring)
 	-- The actual item stack string may be different from what the "giver"
 	-- entered (e.g. big numbers are always interpreted as 2^16-1).
 	stackstring = itemstack:to_string()
+	-- Air is not that useful as an item, but it also does no real harm.
+	local you_hacker = itemstack:get_name() == "air"
```

Comment:
Please remove all 'you hacker' stuff as it's unnecessary code bloat. Then i'll be +1.